docs/elasticsearch.md

```diff
@@ -1,3 +1,120 @@
+| `type`                 | required |                         | Must be `elasticsearch`.                                                                                                                                                |
+| `collection_interval`  | optional | `60s`                   | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`.                                                                                  |
+| `endpoint`             | optional | `http://localhost:9200` | Base URL for the Elasticsearch REST API                                                                                                                                 |
+| `username`             | optional |                         | Username for authentication with Elasticsearch. Required if `password` is set.                                                                                          |
+| `password`             | optional |                         | Password for authentication with Elasticsearch. Required if `username` is set.                                                                                          |
+| `collect_jvm_metrics`  | optional | `true`                  | If true, supported JVM metrics will be collected                                                                                                                        |
+| `skip_cluster_metrics` | optional | `true`                  | If true, cluster level metrics will not be collected. To prevent duplicate metrics, this should be set to false for one node only.                                      |
+| `insecure`             | optional | true                    | Signals whether to use a secure TLS connection or not. If insecure is true TLS will not be enabled.                                                                     |
+| `insecure_skip_verify` | optional | false                   | Whether to skip verifying the certificate or not. A false value of insecure_skip_verify will not be used if insecure is true as the connection will not use TLS at all. |
+| `cert_file`            | optional |                         | Path to the TLS cert to use for mTLS required connections.                                                                                                              |
+| `key_file`             | optional |                         | Path to the TLS key to use for mTLS required connections.                                                                                                               |
+| `ca_file`              | optional |                         | Path to the CA cert. As a client this verifies the server certificate. If empty, uses system root CA.                                                                   |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    elasticsearch_metrics:
```

Comment:
It's preferred for the receiver and pipeline IDs to stay the same as the receiver type in the sample configuration. E.g.  "s/{APP}_metrics/{APP}", "s/{APP}_pipeline/{APP}", "s/{APP}_default/{APP}"

Reply:
Got it, will use this convention going forward.